python/level-t/main.py

```diff
@@ -1,2 +1,9 @@
-def is_palindrome(string):
-    pass
+import string
```

Comment:
In the Agile Software Manifesto, see sule 10: simplicity ­– arte of maximizing amount of work not done – is essential. Using librairies to simplify your work is welcome 👍